data_scrapers/alameda_county.py

```diff
@@ -0,0 +1,167 @@
+    # See: https://developers.arcgis.com/rest/services-reference/layer-feature-service-.htm#GUID-20D36DF4-F13A-4B01-AA05-D642FA455EB6
+    if "dateFieldsTimeReference" in cases_header["editingInfo"] or "editFieldsInfo" in cases_header:
+        raise FutureWarning("A timezone may now be specified in the metadata.")
+    # convert timestamp to datetime object
+    update = datetime.fromtimestamp(timestamp/1000, tz=timezone.utc)
+    out["update_time"] = update.isoformat()
+
+    # get cases, deaths, and demographics data
+    out["series"] = get_timeseries()
+    demo_totals, counts_lt_10 = get_demographics()
+    out["case_totals"], out["death_totals"] = demo_totals["case_totals"], demo_totals["death_totals"]
+    out["meta_from_baypd"] = "These datapoints have a value less than 10: " + ", ".join([item for item in counts_lt_10])
+    return out
+
+
+# Confirmed Cases and Deaths
+def get_timeseries() -> Dict: 
+    """Fetch daily and cumulative cases and deaths by day
+    Returns the dictionary value for "series": {"cases":[], "deaths":[]}.
+    To create a DataFrame from this dictionary, run
+    'pd.DataFrame(get_timeseries())'
+    """
+
+    series = {"cases":[], "deaths":[]} # dictionary holding the timeseries for cases and deaths
+
+    # query API
+    param_list = {'where':'0=0', 'resultType': 'none', 'outFields': 'Date,AC_Cases,AC_CumulCases,AC_Deaths,AC_CumulDeaths', 'outSR': 4326,'orderByField': 'Date', 'f': 'json'}
+    response = requests.get(cases_deaths, params=param_list)
+    response.raise_for_status()
+    parsed = json.loads(response.content)
+    features = [obj["attributes"] for obj in parsed['features']]
+    
+    # convert dates
+    for obj in features:
+        month, day, year = obj['Date'].split('/')
+        if int(month) < 10:
+            month = '0' + month
+        if int(day) < 10:
+            day = '0' + day
+        obj['Date'] = "{}-{}-{}".format(year, month, day)
+
+    # re-key series
+    new_keys = ["date", "cases", "cumul_cases", "deaths", "cumul_deaths"]
+    re_keyed = [dict(zip(new_keys, list(entry.values())))
+                           for entry in features]
+
+    # parse series into cases and deaths
+    death_keys = ["date", "deaths", "cumul_deaths"]
+    case_keys = ["date", "cases", "cumul_cases"]
+    series["cases"] = [{k: v for k, v in entry.items() if k in case_keys} for entry in re_keyed]
+    series["deaths"] = [{k: v for k, v in entry.items() if k in death_keys} for entry in re_keyed]
+
+    return series
+
+def get_notes() -> str:
+    """Scrape notes and disclaimers from dashboards."""
+    notes = ""
+    driver = webdriver.Firefox()
+    driver.implicitly_wait(30)
+    for url in dashboards:
+        driver.get(url)
+        soup = BeautifulSoup(driver.page_source,'lxml')
+        for p_tag in soup.find_all('p'):
+            if 'Notes' in p_tag.get_text():
+                notes += p_tag.get_text().strip()
+    return notes
```

Comment:
I think we need to make sure to call `driver.quit()` before returning here. (You could also use a `try...finally` block to quit.)